tests/scripts/helm.sh

```diff
@@ -83,6 +83,13 @@ install() {
 }
 
 helm_reset() {
+    if ! helm_loc="$(type -p "helm")" || [[ -z ${helm_loc} ]]; then
+        local dist
+        dist="$(uname -s)"
+        # shellcheck disable=SC2021
+        dist=$(echo "${dist}" | tr "[A-Z]" "[a-z]")
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2021

`tr "A-Z" "a-z"` accomplishes the same thing without having to ignore a shellcheck error.

Reply:
After resolving SC2021, there are the following two warnings. So I fixed these as well.

https://github.com/koalaman/shellcheck/wiki/SC2018
https://github.com/koalaman/shellcheck/wiki/SC2019